packages/demand-bloc-quotes/package.json

```diff
@@ -23,16 +23,6 @@
   },
   "dependencies": {
     "@karhoo/demand-api": "^16.12.0",
-    "rxjs": "6.5.5"
-  },
-  "devDependencies": {
-    "@microsoft/api-extractor": "7.7.13",
-    "@types/date-and-time": "0.6.0",
-    "@types/jest": "25.1.3",
-    "@types/uuid": "7.0.2",
-    "jest-fetch-mock": "3.0.3",
-    "ts-jest": "25.2.1",
-    "tsdx": "0.13.1",
-    "typescript": "3.8.3"
+    "rxjs": "^6.6.7"
```

Comment:
There is already rxjs version 7. Did we consider bumping major version?

Reply:
I've tried at the beginning but left with huge amount of TS errors
maybe I should give it another try now because I've managed to have that repo green

Reply:
Let's keep v6 then and bump separately